pages/1.12/monitoring/index.md

```diff
@@ -92,9 +92,13 @@ You can sort system health by `systemd` unit. However, this search can bring up
 
 The system health API relies on Mesos-DNS to know about all the cluster hosts. It finds these hosts by combining a query from `mesos.master` A records as well as `leader.mesos:5050/slaves` to get the complete list of hosts in the cluster. This system has a known bug, in which an agent will not show up in the list returned from `leader.mesos:5050/slaves` if the Mesos agent service is not healthy. This means the system health API will not show this host. If you experience this behavior it is most likely that your Mesos agent service on the missing host is unhealthy.
 
+### Diagnostics bundle contents
+
+The contents of the generated bundle are not stable over time and any internal or third party bundle analysis tooling should be programmed very defensively in this regard.
+
 ## Troubleshooting
 
-If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-d3t.service`).
+If you have any problems, you can check if the diagnostics service is running by SSHing to the Mesos leading master and checking the `systemd` status of the diagnostics component (`dcos-diagnostics.service`).
```

Comment:
@makkes @jdef @janisz Does this/these edits need to be backported to 1.10 and 1.11? Specifically the change to `dcos-diagnostics.service`????